client.go

```diff
@@ -114,10 +115,21 @@ func (c *Client) Run(command string, stdout io.Writer, stderr io.Writer) (int, e
 		return 1, err
 	}
 
-	go io.Copy(stdout, cmd.Stdout)
-	go io.Copy(stderr, cmd.Stderr)
+	var wg sync.WaitGroup
+	wg.Add(2)
+
+	go func() {
+		defer wg.Done()
+		io.Copy(stdout, cmd.Stdout)
```

Comment:
Why not move the ```io.Copy(stderr, cmd.Stderr)``` here and remove the second go routine completely and make ```wg.Add(2)``` be ```wg.Add(1)``` ?